File: app/src/main/java/com/thedesignerx/saim/new_message_app/MainActivity.java
package com.thedesignerx.saim.new_message_app;

import android.os.Bundle;
import android.app.Activity;
import android.app.PendingIntent;
import android.content.Intent;
import android.telephony.SmsManager;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.EditText;
import android.widget.ImageButton;
import android.widget.Toast;

public class MainActivity extends Activity {

    EditText mobileno, message;
    ImageButton sendsms;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mobileno = (EditText) findViewById(R.id.editText1);
        message = (EditText) findViewById(R.id.editTextPassword);
        sendsms = (ImageButton) findViewById(R.id.button1);
        sendsms.setOnClickListener(new OnClickListener() {

            @Override
            public void onClick(View arg0) {
                String no = mobileno.getText().toString();
                String msg = message.getText().toString();
                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
                PendingIntent pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, 0);
                SmsManager sms = SmsManager.getDefault();
                sms.sendTextMessage(no, null, msg, pi,null);

                Toast.makeText(getApplicationContext(), "Message Sent successfully!",
                        Toast.LENGTH_LONG).show();

            }
        });
    }


}

File: app/src/main/java/com/thedesignerx/saim/new_message_app/MainActivity.kt
package com.thedesignerx.saim.new_message_app

import android.app.Activity
import android.app.PendingIntent
import android.content.Intent
import android.os.Bundle
import android.telephony.SmsManager
import android.view.View
import android.widget.EditText
import android.widget.ImageButton
import android.widget.Toast

class MainActivity : Activity() {
    var mobileno: EditText? = null
    var message: EditText? = null
    var sendsms: ImageButton? = null
    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        mobileno = findViewById<View>(R.id.editText1) as EditText
        message = findViewById<View>(R.id.editTextPassword) as EditText
        sendsms = findViewById<View>(R.id.button1) as ImageButton
        sendsms!!.setOnClickListener {
            val no = mobileno!!.text.toString()
            val msg = message!!.text.toString()
            val intent = Intent(applicationContext, MainActivity::class.java)
            val pi = PendingIntent.getActivity(applicationContext, 0, intent, 0)
            val sms = SmsManager.getDefault()
            sms.sendTextMessage(no, null, msg, pi, null)
            Toast.makeText(applicationContext, "Message Sent successfully!",
                    Toast.LENGTH_LONG).show()
        }
    }
}